src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,4 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/create-book-table.yml
+      
```

Comment:
Also here

Reply:
yes, it looks odd because the last line here is actually not empty and contains some spaces